clj-refactor.el

```diff
@@ -3653,7 +3661,7 @@ See: https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-create-fn-from-e
         (buffer-substring (point-min) (point-max)))
     s))
 
-(defun cljr--is-keyword? (s)
+(defun cljr--sans-ns-keyword? (s)
```

Comment:
I see your point.

> The intent everywhere else (than the new usage site added in this PR) is to check for keywords, aliased, fully-qualified, or not.

this is an assumption tho. have you checked all call sites if they break when the bug is fixed? I am happy to check and refactor this as you suggesting outside this PR.